README.md

```diff
@@ -57,7 +63,3 @@ sudo SENTRY_IMAGE=us.gcr.io/sentryio/sentry:83b1380 ./install.sh
 ```
 
 Where you replace `83b1380` with the sha you want to use.
-
-[build-status-image]: https://github.com/getsentry/self-hosted/workflows/test/badge.svg
```

Comment:
Why remove these?

Reply:
They are just references that don't do anything by themselves. I am guessing they were used at some point and then forgotten - this is why I never use them and just inline links.